Source/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -55,22 +53,29 @@ static void SortAccessibilityElements(NSMutableArray *elements)
   [elements sortUsingComparator:comparator];
 }
 
+static CGRect ASAccessibilityFrameForNode(ASDisplayNode *node) {
+  ASDisplayNode *supernode = node.firstNonLayerNode;
+  ASDisplayNodeCAssert(!supernode.isLayerBacked, @"No non-layerbacked node found.");
+
+
+  CALayer *layer = node.layer;
+  return [layer convertRect:node.bounds toLayer:ASFindWindowOfLayer(layer).layer];
```

Comment:
Should we remove `firstNonLayerNode` since it isn't actually used here?

Reply:
done